docs/setup_guides/hpc.rst

```diff
@@ -169,7 +275,10 @@ Copy and paste the following text into a file, dask.sh:
     # >>> client = Client(scheduler_file='~/scheduler.json')
 
     rm -f scheduler.json
-    mpirun --np 12 dask-mpi --nthreads 6 --memory-limit 24e9 --interface ib0
+    mpirun --np 12 dask-mpi \
+        --nthreads 6 \
+        --memory-limit 24e9 \
+        --interface ib0
 
 This script asks for two nodes with 36 cores each. It breaks up each
```

Comment:
In our PBS installation, such a script asks for two nodes with 72 cores each.

Reply:
Fixed above. Thanks for the catch.